<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QInteractorStylePaintBrush</class>
 <widget class="QWidget" name="QInteractorStylePaintBrush">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>177</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PaintBrush</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_25">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#0000a5;&quot;&gt;Current Stage&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Paint brush</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>Brush Size</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSlider" name="BrushSizeSlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>40</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="BrushSizeSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>40</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Brush Style</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QComboBox" name="BrushComBox">
          <item>
           <property name="text">
            <string>Square</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Circle</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Brush Option:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="eraserCheckBox">
        <property name="text">
         <string>Eraser</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(226, 226, 226)</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLabel" name="activeLabelLabel">
           <property name="font">
            <font>
             <family>Arial</family>
            </font>
           </property>
           <property name="text">
            <string>Brush Label:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="labelComboBox">
           <property name="font">
            <font>
             <family>Arial</family>
            </font>
           </property>
           <property name="iconSize">
            <size>
             <width>13</width>
             <height>13</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>No label</string>
            </property>
            <property name="icon">
             <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
              <normaloff>:/icons/transparentbox.png</normaloff>:/icons/transparentbox.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Vessel Wall</string>
            </property>
            <property name="icon">
             <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
              <normaloff>:/icons/bluebox.png</normaloff>:/icons/bluebox.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Lumen</string>
            </property>
            <property name="icon">
             <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
              <normaloff>:/icons/redbox.png</normaloff>:/icons/redbox.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Calcification</string>
            </property>
            <property name="icon">
             <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
              <normaloff>:/icons/greenbox.png</normaloff>:/icons/greenbox.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hemorrhage</string>
            </property>
            <property name="icon">
             <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
              <normaloff>:/icons/yellowbox.png</normaloff>:/icons/yellowbox.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>LRNC</string>
            </property>
            <property name="icon">
             <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
              <normaloff>:/icons/lightbluebox.png</normaloff>:/icons/lightbluebox.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>LM</string>
            </property>
            <property name="icon">
             <iconset resource="../../Main/resources/IADE_Analyzer.qrc">
              <normaloff>:/icons/purplebox.png</normaloff>:/icons/purplebox.png</iconset>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_13">
           <property name="font">
            <font>
             <family>Arial</family>
            </font>
           </property>
           <property name="text">
            <string>Brush Opacity:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QSpinBox" name="opacitySpinBox">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="opacitySlider">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
             <property name="sliderPosition">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Main/resources/IADE_Analyzer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>BrushSizeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BrushSizeSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BrushSizeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BrushSizeSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eraserCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame_4</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>opacitySpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>opacitySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>opacitySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>opacitySpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>57</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
